integration.js

```diff
@@ -43,6 +44,7 @@ beforeEach(() => {
       "reset_tables.sql",
       "reset_roles.sql",
       "add_testkcs.sql",
+      "add_testkcs_stock.sql",
```

Comment:
This does not need to be in the global beforeEach can put it in `process_approval.js`'s beforeEach